slices/bash.yaml

```
@@ -1,10 +1,19 @@
 package: bash
 
 slices:
+  config:
+    contents:
+      /etc/bash.bashrc:
+      /etc/skel/.bash_logout:
+      /etc/skel/.bashrc:
+      /etc/skel/.profile:
+
   bins:
     essential:
+      - bash_config
       - libc6_libs
       - libtinfo6_libs
     contents:
-      # user should manually link /bin/sh to /bin/bash
       /bin/bash:
+      /bin/rbash: { symlink: /bin/bash }
```

```suggestion
      /bin/rbash:
```

The deb already contains the symlink so there's no need to re-define it.